lib/rubocop/cop/lint/unescaped_bracket_in_regexp.rb

```diff
@@ -58,6 +58,8 @@ def on_send(node)
             Regexp::Parser.parse(text.value)&.each_expression do |expr|
               detect_offenses(text, expr)
             end
+          rescue Regexp::Parser::ParserError
+            nil
```

Comment:
Can you replace `nil` with an intentional comment?

```suggestion
            # Does not handle exceptions raised by the regexp_parser gem.
```

Reply:
Done.